app/routes/lending/index.tsx

```diff
@@ -0,0 +1,45 @@
+import { Route, Switch } from 'react-router-dom';
+import LendableObjectDetail from 'app/routes/lending/LendableObjectDetail';
+import LendableObjectEdit from 'app/routes/lending/LendableObjectEdit';
+import LendableObjectsList from 'app/routes/lending/LendableObjectsList';
+import PageNotFound from 'app/routes/pageNotFound';
+import LendableObjectAdminDetail from './LendableObjectAdminDetail';
+import LendableObjectsAdmin from './LendableObjectsAdmin';
+
+const lendingRoute = ({
+  match,
+}: {
+  match: {
+    path: string;
+  };
```

Comment:
I would appreciate if you would use `useRouteMatch` from `react-router-dom` so that there's less refactoring for 
`react-router` v6

```ts
  const { path} = useRouteMatch()
```